dcos_tunnel/cli.py

```diff
@@ -106,6 +108,7 @@
 import paramiko
 
 logger = util.get_logger(__name__)
+util.configure_process_from_environ()
```

Comment:
I think we can remove this one, the new verbose flags should cover this use case.